Dnd.Ddd.Services/Commands/Handlers/RollAbilityScoresCommandHandler.cs

```diff
@@ -33,10 +33,10 @@ public void Handle(RollAbilityScoresCommand command)
 
             Guard.With<InvalidOperationException>()
                 .Against(
-                    !(character is CharacterDraft),
+                    character.IsCompleted(),
                     $"Attempting to roll ability scores on completed character with UiD: {command.CharacterUiD}!");
```

Comment:
See previous comments ;)
```suggestion
Guard.With<InvalidCharacterStateException>().Against(character.IsCompleted(), character.UiD);
```

Reply:
changed